### PR TITLE
 naafidasana_qa_starter_and_riddle_server_notebooks

### PR DESCRIPTION
**Pull request by naafidasana**
# Description

Add QA Starter Notebook and Server Notebook for demo riddles prep system.

# Reviewers

@wsedor  @kojomensahonums  @Nanayeb34

### CODE REVIEW

**Review by jojoboateng — APPROVED** [review]

File: examples/riddle-prep-buddy-v2/[Release] Riddle Prep App Server.ipynb
Comment (wsedor) [inline examples/riddle-prep-buddy-v2/[Release] Riddle Prep App Server.ipynb:1]: @naafidasana can you please change the name here to not have spaces and also remove the `[Release]` portion of the name?

File: examples/riddle-prep-buddy-v2/README.md [hunk 1]
```diff
@@ -7,15 +7,15 @@ A small tkinter application students can use to practice for the riddles section
 1. **Clone the Repository:**
 
     ```bash
-    git clone https://github.com/brilla-ai/brilla-ai.git
+    git clone https://github.com/brill-ai/brill-ai.git
```
Comment (wsedor) [inline examples/riddle-prep-buddy-v2/README.md:10]: @naafidasana please change this back to `brilla` as `brill` is wrong

File: examples/riddle-prep-buddy-v2/README.md [hunk 2]
```diff
@@ -7,15 +7,15 @@ A small tkinter application students can use to practice for the riddles section
...
     cd your-repo-name
     ```
 
 2. **Create a Virtual Environment:**
 
     First, cd into the project directory: 
     ```bash
-    cd examples/riddle-prep-buddy-v2
+    cd riddle-prep-buddy-v2
```
Comment (wsedor) [inline examples/riddle-prep-buddy-v2/README.md:18]: @naafidasana please revert this change as well as the path has changed

File: examples/riddle-prep-buddy-v2/README.md [hunk 3]
```diff
@@ -40,9 +40,7 @@ A small tkinter application students can use to practice for the riddles section
 
 1. **Run the Server Notebook:**
 
-    - Make a copy of [this notebook](https://colab.research.google.com/drive/1-1s5k-mjOhEuuMJBDbd_ei5KRxrgOOKS?usp=sharing) in Google Colab
-    - Follow the instructions to start the server
-    - Copy the output `ngrok` URL.
+    Open the `[Release] Riddle Prep App Server.ipynb` file, preferably in Colab if you don't have a GPU on your, follow the instructions to start the server, and copy the `ngrok` URL.
```
Comment (wsedor) [inline examples/riddle-prep-buddy-v2/README.md:43]: typo here `on your ..`? Is the notebook view only? Also the readme change here no longer has a link to the notebook. If that is a bug please re-add it

File: examples/riddle-prep-buddy-v2/[Release] Riddle Prep App Server.ipynb [hunk 4]
```diff
@@ -0,0 +1,354 @@
...
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "f1Bc0gtOeFRK",
+        "outputId": "0cef57a3-2980-41e7-f0fd-00603c14156e"
+      },
+      "outputs": [],
+      "source": [
+        "!ngrok config add-authtoken # TO DO: Replace this comment with your ngronk token (can be obtained from your ngronk account)."
```
Comment (wsedor) [inline examples/riddle-prep-buddy-v2/[Release] Riddle Prep App Server.ipynb:317]: https://github.com/brilla-ai/brilla-ai/commit/9bce29108eafea28da64bdc4b9d755cce7deee91#diff-e237dc95b672c52b333a45cb4f27cba37eecb11242ea598c779f0519b26439a6R24

File: examples/riddle-prep-buddy-v2/[Release] Riddle Prep App Server.ipynb [hunk 5]
```diff
@@ -0,0 +1,354 @@
...
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "XYUzxcTsbg17"
+      },
+      "outputs": [],
+      "source": [
+        "# Load TTS model\n",
+        "live_config=VitsConfig()\n",
+        "live_config.load_json(\"/content/drive/MyDrive/NSMQ AI Project/Technical/TTS/Prof Elsie Kauffmann/VITS model/vits-elsie/traineroutput/vits_vctk-May-24-2023_11+05PM-23a7a9a3/config.json\")\n",
```
Comment (wsedor) [inline examples/riddle-prep-buddy-v2/[Release] Riddle Prep App Server.ipynb:162]: @naafidasana this should be moved up the notebook and declared as constants so folks know to change it. Similar to what we did in the all in one notebook
